README.md

```diff
@@ -6,6 +6,82 @@ multibot
 
 A programmatic multi-repository helper.
 
+Multibot allows you to invoke actions (`read`, etc.) on 1+ files in 1+
+repositories concurrently. It is meant to be a helper tool for multi-project
+updates, branches, pull requests, etc.
+
+## Install
+
+```sh
+$ npm install multibot
+```
+
+## Usage
+
+```
+A friendly multi-repository robot.
+
+Usage: multibot --action=<string> [options]
+
+Options:
+  --action       Actions to take                               [string] [required] [choices: "read"]
+  --branch-src   Source git branch to read from                         [string] [default: "master"]
+  --branch-dst   Destination git branch to write to                                         [string]
```

Comment:
`--branch-dst` is a future feature, right? If so, maybe mark it as [NOT IMPLEMENTED] or something.